ts-perf/packages/core/src/stringComparer.ts

```diff
@@ -0,0 +1,87 @@
+import * as os from "node:os";
+
+import { Comparer, defaultEqualer, Equaler } from "@esfx/equatable";
+// @ts-ignore
+import C from "@unicode/unicode-15.1.0/Case_Folding/C/symbols.js";
+// @ts-ignore
+import S from "@unicode/unicode-15.1.0/Case_Folding/S/symbols.js";
```

Comment:
Can we get this just via `Intl.Collator` or something, rather than pulling in the tables and DIYing it?

Reply:
`Intl.Collator` doesn't support string hashing. We don't really use it, so I can drop this dependency.

Reply:
It's totally fine to leave this; I was mainly just curious.